AEPMessaging/Sources/MessagingConstants.swift

```diff
@@ -150,6 +146,12 @@ enum MessagingConstants {
                     static let SWIPE_RIGHT = "swipeRight"
                     static let TAP_BACKGROUND = "tapBackground"
                 }
+
+                enum Inbound {
+                    // schema values
+                    static let SCHEMA_FEED_ITEM = "https://ns.adobe.com/personalization/inbound/feed-item"
+                    static let SCHEMA_IAM = "https://ns.adobe.com/personalization/inbound/in-app-message"
```

Comment:
these names have changed slightly - reflected in the wiki

Reply:
Thanks Steve! I'm aware of the change. I'll update these soon as the test feed items get updated (for the surfaces we are testing with). I want to make sure we have a working setup at all times!